src/algorithms/policy_gradient/sac.jl

```diff
@@ -99,12 +99,12 @@ function (p::SACPolicy)(env)
         s = state(env)
         s = Flux.unsqueeze(s, ndims(s) + 1)
         # trainmode:
-        action = evaluate(p, s)[1][] # returns action as scalar
+        action = dropdims(evaluate(p, s)[1], dims=2) # Single action vec, drop second dim
```

Comment:
With this the action will always be returned as a vector. I'm not sure how you would prefer to have it, if scalars should be treated special in some way. But for now this seemed clean and easy to me so I went this way.

Reply:
I agree this way is cleaner. But it's strange to me that the experiment of SAC with pendulum doesn't throw an exception here.

https://github.com/JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl/blob/db5a7929f0683709c1a6f7936ee770bad9e78f17/src/environments/examples/PendulumEnv.jl#L92

Reply:
I call it with `x[1]` through the ActionTransformedEnv in the Experiment.
```julia
        action_mapping = x -> low + (x[1] + 1) * 0.5 * (high - low),
```

Reply:
I see, thanks for your explanation.